tailwind.config.js

```diff
@@ -137,15 +137,11 @@ export default {
               color: 'var(--amber-12)',
             },
             pre: {
-              color: 'var(--sand-12)',
```

Comment:
No longer need to set color and background color here, since all `<pre>` & `<pre><code>` blocks are highlighted by shiki now!